example-mlem-get-started/generate.sh

```diff
@@ -0,0 +1,209 @@
+set -eux
+
+while read var; do
+  [ -z "${!var+x}" ] && { echo "$var is empty or not set"; exit 1; }
+done << EOF
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+HEROKU_API_KEY
+AZURE_STORAGE_CONNECTION_STRING
+EOF
+
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+REPO_NAME="example-mlem-get-started"
+
+BUILD_PATH="$HERE/build"
+REPO_PATH="$BUILD_PATH/$REPO_NAME"
+
+if [ -d "$REPO_PATH" ]; then
+ echo "Repo $REPO_PATH already exists, please remove it first."
+ exit 1
+fi
+
+mkdir -p $BUILD_PATH
+pushd $BUILD_PATH
+if [ ! -d "$BUILD_PATH/.venv" ]; then
+  virtualenv -p python3 .venv
+  source .venv/bin/activate
+  echo '.venv/' > .gitignore
+  pip install -r $HERE/code/src/requirements.txt
+  git clone https://github.com/iterative/mlem.git
+  pip install -e ./mlem
+fi
+popd
+
+source $BUILD_PATH/.venv/bin/activate
+
+
+TOTAL_TAGS=12
+STEP_TIME=100000
+BEGIN_TIME=$(( $(date +%s) - ( ${TOTAL_TAGS} * ${STEP_TIME}) ))
+export TAG_TIME=${BEGIN_TIME}
+export GIT_AUTHOR_DATE=${TAG_TIME}
+export GIT_COMMITTER_DATE=${TAG_TIME}
+tick(){
+  export TAG_TIME=$(( ${TAG_TIME} + ${STEP_TIME} ))
+  export GIT_AUTHOR_DATE=${TAG_TIME}
+  export GIT_COMMITTER_DATE=${TAG_TIME}
+}
+
+export GIT_AUTHOR_NAME="Mikhail Sveshnikov"
+export GIT_AUTHOR_EMAIL="mike0sv@gmail.com"
+export GIT_COMMITTER_NAME="$GIT_AUTHOR_NAME"
+export GIT_COMMITTER_EMAIL="$GIT_AUTHOR_EMAIL"
+
+
+
+mkdir -p $REPO_PATH
+pushd $REPO_PATH
+
+git init -b main
+cp $HERE/code/README.md .
+cp -r $HERE/code/.devcontainer .
+cp $HERE/.gitattributes .
+cp $HERE/code/src/requirements.txt .
+cp $HERE/code/src/*.py .
+echo ".venv" > .gitignore
+git add .
+tick
+git commit -m "Initialize Git repository"
+git tag -a "0-git-init" -m "Git initialized."
+
+
+################## MLEM
+
+git branch simple
+git checkout simple
+
+mlem init
+tick
+git add .mlem
+git commit -m "Initialize MLEM project"
+git tag -a "1-mlem-init" -m "MLEM initialized."
+
+
+python train.py
+git add .mlem
+tick
+git commit -m "Train the model"
+git tag -a "2-train" -m "Model trained."
+
+
+python evaluate.py
+mlem apply rf iris.csv
+echo "sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
+0,1,2,3" > new_data.csv
+mlem apply rf new_data.csv -i --it pandas[csv]
+git add metrics.json
+tick
+git commit -m "Evaluate model"
+git tag -a "3-eval" -m "Metrics calculated"
+
+
+mlem init s3://example-mlem-get-started
+mlem clone rf s3://example-mlem-get-started/rf
+
+
+mlem declare builder pip pip_config -c target=build/ -c package_name=example_mlem_get_started
+git add .mlem
+tick
+git commit -m "Add package config"
+git tag -a "4-pack" -m "Pip package config added"
+
+mlem declare env heroku staging
+mlem declare deployment heroku myservice -c app_name=example-mlem-get-started-app -c model=rf -c env=staging
+git add .mlem
+tick
+git commit -m "Add env and deploy meta"
+git tag -a "5-deploy-meta" -m "Target env and deploy meta added"
+
+if heroku apps:info example-mlem-get-started-app; then
```

Comment:
now, we need to change it?

Reply:
Yes, doing that in https://github.com/iterative/mlem.ai/pull/131